### PR TITLE
Redesigned Logger Webhooks & Audit Webhooks pages

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

- Added the capability to set custom names to logger & audit webhooks
- Added the capability to edit webhooks
- Added the capability to delete wehooks
- Display webhook status

*NOTE:* Add certificates is pending as we need to define how we can store this information

## How does it look?

Screenshot 2023-01-31 at 20 01 05
Screenshot 2023-01-31 at 20 01 00
Screenshot 2023-01-31 at 19 59 28
Screenshot 2023-01-31 at 19 59 23
Screenshot 2023-01-31 at 19 59 17
Screenshot 2023-01-31 at 19 59 08
Screenshot 2023-01-31 at 19 58 59

Signed-off-by: Benjamin Perez <benjamin@bexsoft.net>

### CODE REVIEW

**Comment by prakashsvmx** [comment]
@bexsoft  some observations: 
1. Is Auth Token field mandatory ? . i think it should be optional.
2. While adding new webhook, the field widths are not equal
<details>

image

</details>
3. Unable to reset/disbale the default audit_webhook

<details>

https://user-images.githubusercontent.com/23444145/216042671-3d9d37fb-4ee5-4245-98b0-873e86e825be.mp4

</details>

**Comment by bexsoft** [comment]
> 

Fixed:

Screenshot 2023-02-01 at 13 49 11
Screenshot 2023-02-01 at 13 48 56
Screenshot 2023-02-01 at 13 48 49

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM.

**Comment by prakashsvmx** [comment]
@bexsoft  could you please resolve the conflicts ?

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/NotificationEndpoints/WebhookSettings/EditWebhookEndpoint.tsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  const updateWebhook = () => {
+    if (saving) {
+      return;
+    }
+
+    if (invalidInputs.length !== 0) {
+      return;
+    }
+
+    if (endpoint.trim() === "") {
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/NotificationEndpoints/WebhookSettings/EditWebhookEndpoint.tsx:114]: Click on Default Endpoint, 
toggle Enabled On 
Do not enter value of Endpoint
Click Save. this throws JS error.  ( validation error is not shown for mandatory field)

```
react-dom.development.js:4312 Uncaught TypeError: Cannot read properties of undefined (reading 'trim')
```

**Review by prakashsvmx — CHANGES_REQUESTED** [review]
Tested Changes look good to me :+1:  

Suggestion : The Edit option could  have explicit button

File: portal-ui/src/screens/Console/NotificationEndpoints/WebhookSettings/EditWebhookEndpoint.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/NotificationEndpoints/WebhookSettings/EditWebhookEndpoint.tsx]: Fixed this issue, For the edit option will send it in another PR as there is a small bug inDataTable from mds that needs to be fixed first

**Review by reivaj05 — APPROVED** [review]

**Review by jinapurapu — APPROVED** [review]

**Review by allanrogerr — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]
Thank you.  LGTM :+1:

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**